### ISSUE TITLE
Pixel Launcher crashes endlessly on 14 August release

### ISSUE THREAD

**Issue by 1987tl** [MisterZtr/PixelOS_gsi#104]
I just updated to the PixelOS 14 August release and the Pixel Launcher application crashes immediately once booting up the system and contines to crash endlessly, even when using a custom launcher. As a result, you cannot switch between apps or even navigate to the home screen. Clearing the application data and cache of the Pixel Launcher didn't help.

My device is Samsung Galaxy Tab S6 and I am using image PixelOS_treble_arm64_bN_slim-14.0-20240824.img.xz

Thankfully, you can swipe down the top navigation drawer and access the system settings. From here, you can still open up any other applications such as the browser.

The workaround for me is to install/downgrade an earlier version of the Pixel Launcher apk. This one works for me:
https://www.apkmirror.com/apk/google-inc/pixel-launcher/pixel-launcher-14-release/pixel-launcher-14-12-android-apk-download/

Although all the builds are labelled as version 14 (907), they do seem to be different. The one uploaded to APKMirror on August 17 crashes, but the August 7 one (linked above) works fine.

Other than this, the August release is working great on my device.

**Comment by KyonLi** [comment]
Same here, [crash log](https://bin.mkr.pw/~66cb0148632ce757b6264b3a)

**Comment by jaimbo** [comment]
Yep, confirming same issue and the suggested Aug 7th build APK got things working for me again

**Comment by zslingbgbwxhn** [comment]
same，when smallest width >=600(tablet view) pixel launcher will crash

**Comment by ankan2008** [comment]
> I just updated to the PixelOS 14 August release and the Pixel Launcher application crashes immediately once booting up the system and contines to crash endlessly, even when using a custom launcher. As a result, you cannot switch between apps or even navigate to the home screen. Clearing the application data and cache of the Pixel Launcher didn't help.
> 
> My device is Samsung Galaxy Tab S6 and I am using image PixelOS_treble_arm64_bN_slim-14.0-20240824.img.xz
> 
> Thankfully, you can swipe down the top navigation drawer and access the system settings. From here, you can still open up any other applications such as the browser.
> 
> The workaround for me is to install/downgrade an earlier version of the Pixel Launcher apk. This one works for me: https://www.apkmirror.com/apk/google-inc/pixel-launcher/pixel-launcher-14-release/pixel-launcher-14-12-android-apk-download/
> 
> Although all the builds are labelled as version 14 (907), they do seem to be different. The one uploaded to APKMirror on August 17 crashes, but the August 7 one (linked above) works fine.
> 
> Other than this, the August release is working great on my device.

can u explain how u installed custom launcher, or even downgraded pixel launcher.
I am also facing the same issue in my lenovo tab k10, but i cant access a single app

**Comment by ankan2008** [comment]
> same，when smallest width >=600(tablet view) pixel launcher will crash

I tried putting dpi to 440 in my tab, yet the launcher crashed frequently

**Comment by zslingbgbwxhn** [comment]
> > same，when smallest width >=600(tablet view) pixel launcher will crash
> 
> I tried putting dpi to 440 in my tab, yet the launcher crashed frequently

you can access setting through quick panel and into application -> chrome or any browser -> download pixel launcher for android14 release at  August 7 ->install
